sources/packages/backend/apps/api/src/route-controllers/config/config.controller.ts

```diff
@@ -38,6 +39,7 @@ export class ConfigController extends BaseController {
       maximumIdleTimeForWarningAEST:
         this.configService.maximumIdleTimeForWarningAEST,
       appEnv: this.configService.appEnv,
+      queueDashboardURL: `${process.env.QUEUE_DASHBOARD_BASE_URL}/${BULL_BOARD_ROUTE}`,
```

Comment:
For the base URL, can we use the current context URL similar to the way how we get API URL in web?

![image](https://github.com/user-attachments/assets/5021a1de-4ecb-4390-965c-740ce292ce58)

Reply:
The above-mentioned one has two hardcoded values (including the port) while the one in the PR is fully configurable. I would change the one in the example to follow the same idea from the one in this PR 😄 
![image](https://github.com/user-attachments/assets/5151a464-75a9-4462-ba6a-0df2a5de369c)